cli/auth.go

```diff
@@ -214,7 +214,10 @@ func (a *authFlow) makeDeviceCodeRequest(ctx context.Context, discovery *openIDD
 }
 
 func (a *authFlow) directUser(code *deviceCodeResponse) error {
-	fmt.Fprintf(a.console, "To authorize this device, visit:\n\t%s\n", code.VerificationURIComplete)
+	fmt.Fprintf(a.console, `You can log into Viam through the opened browser window or follow the URL below;
+simply confirm the code in the URL matches the one shown in your browser.
```

Comment:
Nice; changed the language to the above. I'd be in favor of changing `viam auth` to `viam login` but that would be breaking; what do you think @cheukt ? Otherwise I'll just add `viam login` as an alias.

Reply:
I opted to switch `viam auth` to `viam login` and maintain `auth` as an alias for `login`. Happy to revert that and just keep `auth` for now, up to you @cheukt. IMO, as @andf-viam mentioned, `login` is a better "opposite" to `logout` and matches the language on the app.

Reply:
I like login as well 👍